ui/vite.config.ts

```diff
@@ -34,5 +34,9 @@ export default defineConfig(({ mode }) => {
                 },
             },
         },
+        test: {
+            globals: true,
+            environment: 'happy-dom'
```

Comment:
_(no need to change anything happy-dom is a great choice I'm just adding a remark here)_
I guess you checked out the different environments but if not then then keep in mind that if you run into compatibility issues then you might have to switch to jsdom
A rough rule of thumb is that happy-dom might be faster (depending on your use case) but jsdom might be more complete (simulates more of what a browser would).

Reply:
Thanks, I'll keep that in mind! :)